examples/outcomes/1/outcomes2.json

```diff
@@ -1,9 +1,7 @@
 {
```

Comment:
can you rename them all? it's a little weird to have outcomes2 and 3 but not the rest of the numbers?